spec/integration/probes/elasticsearch_spec.rb

```diff
@@ -24,17 +24,21 @@
     end
 
     it "handles uninitialized probe dependencies" do
+      # Temporarily uninstall NetHTTP and HTTPClient probes
       TempNetHTTPProbe    = Skylight::Probes::NetHTTP::Probe
       TempHTTPClientProbe = Skylight::Probes::HTTPClient::Probe
       Skylight::Probes::NetHTTP.send(:remove_const, :Probe)
       Skylight::Probes::HTTPClient.send(:remove_const, :Probe)
+      allow_any_instance_of(::Net::HTTP).to receive(:request){|obj, *args| obj.request_without_sk(*args)}
+      allow_any_instance_of(::HTTPClient).to receive(:do_request){|obj, *args| obj.do_request_without_sk(*args)}
```

Comment:
I'm trying to test the case where the NetHTTP and HTTPClient probes have not been enabled in Elasticsearch. To do this, I'm removing their constants and un-doing the monkey-patch changes which those probes make to their respective classes.

Another way to test this would be to setup ES tests in such a way that those probes aren't installed, but it doesn't look like that can be done on a per-test basis; once installed a probe is active for the entire test suite. (We need to test both the cases where they ARE and ARE NOT installed).